src/client/components/Inputs/Select/hooks/useComponents.tsx

```diff
@@ -17,7 +17,7 @@ import { SelectProps } from 'client/components/Inputs/Select/types'
 type Returned = ReactSelectProps['components']
 
 export const useComponents = (props: SelectProps): Returned => {
-  const { collapsibleGroups, isMulti, multiLabelSummaryKey, selectableGroups } = props
+  const { collapsibleGroups, inputHidden, isMulti, multiLabelSummaryKey, selectableGroups } = props
```

Comment:
This prop is passed to the selectComponents.Input `isHidden` prop. There isn't a Select prop that is available that passes it directly to the internal input. 

I added it because the default behavior of react-select is to hide the input when selecting an option:
 
https://github.com/user-attachments/assets/6d88a7ce-ffaf-4dd5-bbf9-ca5f2cd2d602



Passing this prop as false to the select input makes it visible al the time. 

(related issue for reference: https://github.com/JedWatson/react-select/issues/3068)